src/Paysera/Bundle/CodeGeneratorBundle/Service/DefinitionDecorator.php

```diff
@@ -15,13 +15,15 @@
 
```

Comment:
Only `use Raml\ApiDefinition;` and `use Paysera\Bundle\CodeGeneratorBundle\Entity\Definition\ApiDefinition as DecoratedDefinition;` are required